projects/cloud_ml/scripts/scoring.py

```diff
@@ -0,0 +1,100 @@
+import sys
+sys.path.append("/home/yakelyn/explore/yakelyn/utils/")
+import mycolors
+import os
+import fv3viz
+import xarray as xr
+import intake
+import fsspec
+from dask.diagnostics import ProgressBar
+from vcm.catalog import catalog as CATALOG
+from vcm.fv3.metadata import standardize_fv3_diagnostics
+from vcm.safe import get_variables
+from vcm import convert_timestamps
+import cftime
+import numpy as np
+from matplotlib.gridspec import GridSpec
+from matplotlib.colors import LogNorm
+import matplotlib.pyplot as plt
+from cartopy import crs as ccrs
+from sklearn.metrics import mean_squared_error
+from math import sqrt
+import matplotlib.dates as mdates
+import matplotlib.ticker as ticker
+import vcm
+
+def plot_global_map_biases(
+    ds_a,
+    ds_b, 
+    GRID,
+    var_name,
+    cmap,                    
+    units,
+    output_dir,
+    plot_filename,
+    title,
+    title_a,
+    title_b):
+
+    "Plot a 3 row figure. Global maps for ds_a, ds_b, bias = ds_a - ds_b "
+    vmin = np.nanmin(ds_a).round()
```

Comment:
This is up to you, but `fv3viz.plot_cube` already sets the colorscale min and max based on the data's min and max (actually, 2nd and 98th percentile) values, so you could skip passing these values to it. Or better yet, you could make them and `cmap` optional arguments, i.e., 
```
def plot_global_map_biases(..., **plot_cube_kwargs):
    ...
    fv3viz.plot_cube(..., **plot_cube_kwargs)
```
so that you can change them to something else from the top level if the situation calls for it.